.travis.yml

```diff
@@ -1,53 +1,140 @@
-sudo: required
-dist: trusty
+sudo: false
 language: php
 php:
-    - 5.6
-    - 7
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+addons:
+  apt:
+    packages:
+      - graphviz
+      - gdb
 env:
-    global:
-        - CORE_BRANCH=stable9
-    matrix:
-        - DB=pgsql
+  global:
+    - CORE_BRANCH=master
+    - APP_NAME=mediametadata
+    - GH_REF: github.com/interfasys/mediametadata.git
+    - secure: "cLmMqrbVZrvn2GAoVs6JWa5h70tzrAISjPZUfENjl0KYT6B84z3B94W/dGU+I2mxwvyBqEP+x83Bih2CxEflqc7tJmdqC+MM/ZZiigPX7RCjq5XHbzUpPpMbC8MFxT2K3jcFXUitfwZvNaZXJIiK3ZQJU4ayKaegLvI91x1SqH0="
```

Comment:
You can remove GH_REF and secure. Those are needed by some external tools and when generating documentation